### PR TITLE
Chore: cooler env0 logo box

### PR DESCRIPTION
**Pull request by yaronya**
image

### CODE REVIEW

**Comment by avnerenv0** [comment]
Sorry bro but the brand is with small letters 
`ENV0` -> `env0`

**Comment by yaronya** [comment]
@avnerenv0 have a look now 😎

File: node/src/env0-deploy-cli.js [hunk 1]
```diff
@@ -35,23 +35,20 @@ const availableCommands = {
...
-     ///((    **********,,,*      .%%*               .%%,        %%%       %%% /%%        %%%         %%%
-     ///((    **********,,,*       %%%               .%%,        %%%        %%.%%*        (%%#       %%%,
-     ///((    **********,,,*        .%%%%%%%%%%%%    .%%,        %%%        /%%%%           #%%%%%%%%%#
-                                                                                                             `;
+                                  .oooo.   
+                                 d8P'\`Y8b  
+ .ooooo.  ooo. .oo. oooo    ooo 888    888 
+d88' \`88b \`888P"Y88b \`88.  .8'  888    888 
+888ooo888  888   888  \`88..8'   888    888 
+888    .o  888   888   \`888'    \`88b  d88'
```
Comment (avnerenv0) [inline node/src/env0-deploy-cli.js:43]:
```suggestion
888        888   888   \`888'    \`88b  d88' 
```

Comment (yaronya) [reply node/src/env0-deploy-cli.js]: 60cef44

**Pull request merged**